i18n/en_US.csv

```diff
@@ -101,7 +101,7 @@ Region,Region
 "Synced to TaxJar","Synced to TaxJar"
 "Sync Transactions","Sync Transactions"
 "Sync to TaxJar","Sync to TaxJar"
-"SmartCalcs by TaxJar","SmartCalcs by TaxJar"
+"TaxJar API","TaxJar API"
```

Comment:
@ryanreeves-taxjar Should this be `TaxJar Sales Tax Automation` instead?

Reply:
Good catch!  Yes it should.